app/models/miq_cim_derived_metric.rb

```diff
@@ -1,7 +0,0 @@
-class MiqCimDerivedMetric < ApplicationRecord
-  belongs_to    :miq_storage_metric
-
-  # acts_as_list was blindly upgraded from "~>0.1.4" to "~>0.7.2" to fix
-  # a rails 4.x issue.  Keep this in mind when we try using acts_as_list.
-  acts_as_list  :scope => :miq_storage_metric
```

Comment:
I *think* this is the last caller of acts_as_list, so I think we can remove the gem... @jrafanie?